packages/chaire-lib-common/src/services/geodata/ManageOverlappingLines.ts

```diff
@@ -1,58 +1,48 @@
-import { lineOffset, lineOverlap, lineString, LineString } from '@turf/turf';
+/*
+ * Copyright 2022, Polytechnique Montreal and contributors
+ *
+ * This file is licensed under the MIT License.
+ * License text available at https://opensource.org/licenses/MIT
+ */
+import { lineOffset, lineOverlap, lineString } from '@turf/turf';
+import { LineString } from 'geojson';
+
+export const OFFSET_WIDTH = 3; // Offset of each line in meters
 
 interface OverlappingSegments {
+    /**
+     * GeoJSON data containing the overlapping coordinates
+     */
     geoData: GeoJSON.Feature<LineString>;
+    /**
+     * IDs of each line in conflict with the specified overlap
+     */
     crossingLines: number[];
+    /**
+     * Directions of each line in conflict.
+     * True means the line has the overlap in the same coordinates order as geodata.
+     * False means the line has the overlap is in the opposite direction.
+     */
     directions: boolean[];
 }
 
-export const manageOverlappingLines = (
+/**
+ * Offset overlapping lines when multiple lines use the same road or segment.
+ * This is to be able to visually follow where each line goes.
+ * @param layerData GeoJSON data containing the lines to offset (will be modified)
+ * @returns Same GeoJSON data as entry with the offsetted lines
```

Comment:
But if the layerData will be modified and this is the same geojson data as entry, is it useful to have the return value?